client/components/telecom/telephony/group/number/feature/ovhPabx/menu/edit/telephony-group-number-feature-ovh-pabx-menu-edit.html

```diff
@@ -200,42 +200,38 @@ <h2 data-ng-if="['greetSound', 'invalidSound', 'greetSoundTts', 'invalidSoundTts
             <!-- SOUND TYPE SELECTION -->
             <section data-ng-if="$ctrl.menuCtrl.popoverStatus.rightPage === 'greetSoundType' || $ctrl.menuCtrl.popoverStatus.rightPage === 'invalidSoundType'">
                 <div class="section-group">
-                    <label class="section-item item-choice full-width pointer dtable"
+                    <label class="section-item item-choice w-100 dtable"
                            data-ng-if="$ctrl.menuCtrl.popoverStatus.rightPage === 'invalidSoundType'">
-                        <div class="dtr">
-                            <div class="dtd vertical-middle radio-content">
-                                <flat-radio>
-                                    <input type="radio"
-                                           name="soundTypeChoice"
-                                           data-ng-model="$ctrl.model[$ctrl.menuCtrl.popoverStatus.rightPage === 'greetSoundType' ? 'greetSoundType' : 'invalidSoundType']"
-                                           data-ng-value="'none'"
-                                           data-ng-change="$ctrl.onSoudTypeChange()" />
-                                </flat-radio>
-                            </div>
-                            <div class="dtd vertical-middle">
-                                <span class="block"
-                                      data-translate="telephony_number_feature_ovh_pabx_menu_options_sound_type_none">
-                                </span>
-                            </div>
+                        <div class="d-table-cell valign-middle radio-content">
+                            <flat-radio>
```

Comment:
We have to remove `<flat-radio>` directive.

Reply:
in a future time